packages/vitaminjs-build/package.json

```diff
@@ -17,7 +17,10 @@
     "url": "https://github.com/Evaneos/vitaminjs.git"
   },
   "bin": {
-    "vitamin": "bin/vitamin.js"
+    "vitamin-build": "bin/vitamin-build.js",
+    "vitamin-clean": "bin/vitamin-clean.js",
+    "vitamin-start": "bin/vitamin-start.js",
+    "vitamin-test": "bin/vitamin-test.js"
```

Comment:
miss `vitamin-version` ? (just asking)

Reply:
`vitamin -v` is automatically generated by commander which we use in the executable.